stackstate_checks_base/stackstate_checks/base/checks/v2/types.py

```diff
@@ -0,0 +1,15 @@
+from schematics import Model
+from typing import Any, Dict, TypeVar
+
+InstanceType = TypeVar('InstanceType', Model, Dict[str, Any])
+StateType = TypeVar('StateType', Model, Dict[str, Any])
+
+
+class CheckResponse(object):
+    """
+    CheckResponse is the return type of the check function of Agent V2 checks
+    """
+    def __init__(self, *args, **kwargs):
```

Comment:
A learning question: what is the advantage of having `*args`, `**kwargs` instead of named arguments with `None` as default? Like:
```python
def __init__(self, persistent_state = None, transactional_state = None, check_error = None):
```